events.md

```diff
@@ -0,0 +1,130 @@
+# Events
+
+Kubecost emits events when certain things happen. Those events are also recorded in an event log in the interest of diagnosing problems.
+
+## Event kinds
+
+* `AllocationSetSavedEvent` describes a saved AllocationSet
```

Comment:
did you consider going the next level of details on describing the verbs, e.g. save, load, aggregate? i like how simple this is, but when looking at save for example, i'd have question about what exactly this represents. is this save to disk? memory? would backup influence this?